src/onegov/town6/layout.py

```diff
@@ -2144,7 +2144,11 @@ def thumbnail_file_id(self, entry):
         thumbnail = self.thumbnail_field_id
         if not thumbnail:
             return
-        return (entry.values.get(thumbnail) or {}).get('data', '').lstrip('@')
+        if thumbnail in entry.values:
+            item = entry.values[thumbnail]
+            if 'data' in item:
+                return item['data'].lstrip('@')
+        return
```

Comment:
```suggestion
        value = entry.values.get(thumbnail)
        if isinstance(value, list) and value:
            value = value[0]            
        if not value:
            return
        return value.get('data', '').lstrip('@')
```
It probably makes sense to use the first picture as a thumbnail if it's a multiple file field.